src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -135,7 +136,20 @@ const QuickMarcEditor = ({
     handleSubmit(e).then((updatedRecord) => {
       handleSubmitResponse(updatedRecord);
     });
-  }, [deletedRecords, handleSubmit, handleSubmitResponse]);
+  }, [deletedRecords.length, handleSubmit, handleSubmitResponse]);
+
+  const handleFooterSave = useCallback((e, isKeepEditing = false) => {
```

Comment:
What does `footer save` mean? Is it a form saving? If yes then maybe `onFormSubmit` will be more readable